plugins/pipelines/create.js

```diff
@@ -78,7 +86,7 @@ module.exports = () => ({
                                         return pipelineFactory.create(pipelineConfig);
                                     })
                                     // get the default collection for current user
-                                    .then(pipeline =>
+                                    .then(pipeline => {
                                         collectionFactory
```

Comment:
if you're going to add the brace (`{`) in the line above, it'd be safer to use `return` here
```suggestion
                                        return collectionFactory
```